htlcswitch/interceptable_switch.go

```diff
@@ -55,6 +55,10 @@ type InterceptableSwitch struct {
 	// holdForwards keeps track of outstanding intercepted forwards.
 	holdForwards map[channeldb.CircuitKey]InterceptedForward
 
+	// cltvRejectDelta defines the number of blocks before the expiry of the
+	// htlc where we no longer intercept it and instead cancel it back.
+	cltvRejectDelta uint32
```

Comment:
We already have a value for this in `OutgoingCltvRejectDelta`, so no need to make a new value here.

Reply:
Yes, `OutgoingCltvRejectDelta` is passed around in several structures, but afaik it isn't directly accessible here. Or do you suggest to hardcode it to `lncfg.DefaultOutgoingCltvRejectDelta` here?

Reply:
Was thinking that we should use `DefaultOutgoingCltvRejectDelta` but then saw that we use the final CLTV delta here, which I think is more apt as in most cases the intercepted HTLC will be terminated by the program operating the HTLC interceptor.